testfiles/t003.lvt

```diff
@@ -0,0 +1,26 @@
+\documentclass{article}
+\usepackage{fancyvrb}
+\usepackage{tabu}
+\newenvironment{yyyyyy}{START}{END}
+\input{regression-test}
+\begin{document}
+
+\START
+\begin{tabu*}to\linewidth{|XX|}
+% if }{ in the short verb then already not working in tabu 2.09
+A short verb: \verb|$&\~{}|&
+\begin{Verbatim}[commandchars={\\\{\}}]
+And \textit{this} is a \textbf{\textrm{complete}}
+Verbatim environment $&\~\textbraceleft\textbraceright 
```

Comment:
I see now this should not be `\~` but `~` of the test because Verbatim here is configured to have the backslash normal, so this does not check `~` is sanitized.